scripts/dotnet-cli-build/Utils/Monikers.cs

```diff
@@ -9,6 +9,9 @@ namespace Microsoft.DotNet.Cli.Build
     public class Monikers
     {
         public const string SharedFrameworkName = "Microsoft.NETCore.App";
```

Comment:
This looks great.

@richlander @blackdwarf @Petermarcu this is the branding that is now plubmed through the CLI.